packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -2096,6 +2099,7 @@ export const DEFAULT_COMPOSITE_ICONS: {
     CaptionsIcon: JSX.Element;
     CaptionsOffIcon: JSX.Element;
     CaptionsSettingIcon: JSX.Element;
+    PersonIcon: JSX.Element;
```

Comment:
Let's not make the name so generic, if that is the icon f or the ChangeSpokenLanguage, then we call it ChangeSpokenLanguageIcon or something